.dockerignore

```diff
@@ -10,7 +10,8 @@ mesg-dev
 .git
 **/*.test
 **/debug
-node_modules
+**/node_modules
+bin/cli
```

Comment:
Those 2 modifications reduce the data send to docker from 100MB to 24MB

Reply:
are we sure about the _bin/_ to _bin/cli_ because _bin/_ by itself should ignore the whole dir recursively

Reply:
Maybe add also
- docs/
- scripts/
- docker-images/

Reply:
bin/core needs to be send to docker because we are building the development core locally and then copy paste it in docker